nucleus.go

```diff
@@ -16,6 +16,7 @@ type DNA struct {
 	Name                      string
 	Properties                map[string]string
 	PropertiesSchema          string
+	PropertiesSchemaFile      string
```

Comment:
So we I designed this I originally did have a file name in the DNA, but that got pulled out, because we wanted the DNA to not have to know about or refer to any form of storage.  The actual value goes into the DNA, from a file, from a database from wherever.  Then service.go implements how that value gets stored.

I feel ok if this fixes the cryptic error message in this case because aren't going forward with this codebase, but I do think that principle still applies and we don't want to get the file names into the DNA proper, though they could be stored in DNA package.

Reply:
@zippy just for clarification, the error is:

- if you do not provide any schema file OR
- you do provide a schema file but it isn't equal to `properties_schema.json`

then you cannot run `hcdev web` at all because internally the clone fails to copy a file that load then attempts to run but cannot find

this behaviour is wrong whether or not the DNA property is deprecated because it means you _must_ provide a property that only has one valid value :P

it's like being forced to specify `true = true` every time...